src/node/performance/worker_service_impl.js

```diff
@@ -36,6 +36,11 @@
 var BenchmarkClient = require('./benchmark_client');
 var BenchmarkServer = require('./benchmark_server');
 
+exports.quitWorker = function quitWorker(call, callback) {
+  callback(null, {});
+  process.exit(0);
```

Comment:
This looks basically right, but exiting like that may leave some calls hanging. If that's OK, then this is fine.